etc/flameshot.profile

```diff
@@ -0,0 +1,38 @@
+# Firejail profile for flameshot
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/flameshot.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-interpreters.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+include /etc/firejail/disable-xdg.inc
+
+caps.drop all
+ipc-namespace
+netfilter
+no3d
+# nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+
+disable-mnt
+private-cache
+private-dev
```

Comment:
Does the line `private-etc fonts` still work?

Reply:
Getting SSL handshake errors when trying to upload images to imgur; adding hostname, hosts and resolv.conf doesn't seem to fix it - any idea?

Reply:
Works fine now; added and committed.

Reply:
Excellent! Did it require all the etc/ directories (that is, can you remove any of the private-etc options and it still work)?

Reply:
All that it needed was  'private-etc fonts,ca-certificates,ld.so.conf,resolv.conf,ssl'